app/main.py

```diff
@@ -1 +1,34 @@
-# write your code here
+class Animal:
+    alive = []
+
+    def __init__(self, name: str) -> None:
+        self.name = name
+        self.health = 100
+        self.hidden = False
+        self.__class__.alive.append(self)
+
+    def __repr__(self) -> str:
+        return f"{{Name: {self.name}, " \
+               f"Health: {self.health}, " \
+               f"Hidden: {self.hidden}}}"
+
+    def die(self) -> None:
+        self.__class__.alive.remove(self)
+
+
+class Carnivore(Animal):
+    def bite(self, herbivore: bool) -> None:
+        if isinstance(herbivore, Carnivore) or herbivore.hidden:
+            return
```

Comment:
no need to use blank return here, u can just make a condition to use the code below